e2e/app/element/base-element.ts

```diff
@@ -29,10 +29,17 @@ export default class BaseElement extends Container {
    * @param {WaitForSelectorOptions} waitOptions
    */
   async waitForXPath(waitOptions: WaitForSelectorOptions = {visible: true}): Promise<this> {
-    if (this.element === undefined) {
-      this.element = await this.page.waitForXPath(this.xpath, waitOptions);
+    try {
+      if (this.element === undefined) {
+        this.element = await this.page.waitForXPath(this.xpath, waitOptions);
+      }
+      return this;
+    } catch (err) {
+      console.error(`waitForXpath() encountered ${err}`);
+      // Debugging pause
+      // await jestPuppeteer.debug();
+      throw err;
```

Comment:
If someone needs to pause test execution and look for why errored on page,  simply uncomment `await jestPuppeteer.debug();` to allow the inspection of page manually.

Log error if errored. so I know it is related to the screenshot. 
Example:
<img width="1065" alt="Screen Shot 2020-06-18 at 8 50 38 PM" src="https://user-images.githubusercontent.com/35533885/85088032-118af780-b1ad-11ea-9434-ea3b9b5702ed.png">